CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 0.3.0 - 2022-07-12
+
+### Added
+- New `structure` option with default `flat` file structure or `legacy` nested folders.
```

Comment:
Likewise should we be mentioning `default` here since it varies between templates? Also it doesn't look like `flat` is an option anymore?

Reply:
Updated